dao-utils/src/parts/variable.rs

```diff
@@ -25,9 +26,14 @@ impl<T: Default + FromBytes + ToBytes + CLTyped> Variable<T> {
         }
     }
 
-    /// Read from the storage or return default value.
-    pub fn get(&self) -> T {
-        get_key(&self.name).unwrap_or_default()
+    /// Read from the storage or return none
+    pub fn get(&self) -> Option<T> {
+        get_key(&self.name)
+    }
+
+    /// Read from the storage or revert
+    pub fn get_or_revert(&self) -> T {
+        get_key(&self.name).unwrap_or_revert()
```

Comment:
Do we want to just revert without any specific error? I encounterd this many times in this pr. Honestly, it's fine to me, but what if we revert with some generic error in such cases?